src/nvim/testdir/test_cmdline.vim

```diff
@@ -78,26 +78,45 @@ func Test_map_completion()
   call feedkeys(":map <silent> <sp\<Tab>\<Home>\"\<CR>", 'xt')
   call assert_equal('"map <silent> <special>', getreg(':'))
 
+  map <Middle>x middle
+
   map ,f commaf
   map ,g commaf
+  map <Left> left
+  map <A-Left>x shiftleft
   call feedkeys(":map ,\<Tab>\<Home>\"\<CR>", 'xt')
   call assert_equal('"map ,f', getreg(':'))
   call feedkeys(":map ,\<Tab>\<Tab>\<Home>\"\<CR>", 'xt')
   call assert_equal('"map ,g', getreg(':'))
+  call feedkeys(":map <L\<Tab>\<Home>\"\<CR>", 'xt')
+  call assert_equal('"map <Left>', getreg(':'))
+  call feedkeys(":map <A-Left>\<Tab>\<Home>\"\<CR>", 'xt')
+  call assert_equal("\"map <A-Left>\<Tab>", getreg(':'))
   unmap ,f
   unmap ,g
+  unmap <Left>
+  unmap <A-Left>x
 
   set cpo-=< cpo-=B cpo-=k
   map <Left> left
   call feedkeys(":map <L\<Tab>\<Home>\"\<CR>", 'xt')
   call assert_equal('"map <Left>', getreg(':'))
+  call feedkeys(":map <M\<Tab>\<Home>\"\<CR>", 'xt')
+  " call assert_equal("\"map <M\<Tab>", getreg(':'))
```

Comment:
I assume this depends on an unsupported `cpo` value so it's commented out. Same goes for the other assertion.

Reply:
Right, the `<` 'cpo' flag was removed.